frontend/src/components/RepresentativeTable.tsx

```diff
@@ -0,0 +1,39 @@
+import { Trans } from 'react-i18next';
+import classNames from 'classnames';
+import StarSharpIcon from '@material-ui/icons/StarSharp';
+import { RoadmapUser } from '../redux/roadmaps/types';
+import { RoleType } from '../../../shared/types/customTypes';
+import { BusinessIcon } from './RoleIcons';
+import { table, TableRow } from './Table';
+import { userSort, UserSortingTypes } from '../utils/SortRoadmapUserUtils';
+import css from './RepresentativeTable.module.scss';
+
+const classes = classNames.bind(css);
+
+const RepresentativeRow: TableRow<RoadmapUser> = ({ item: user, style }) => (
+  <div style={style} className={classes(css.virtualizedTableRow)}>
+    <div className={classes(css.memberIcon)}>
+      {user.type === RoleType.Admin && <StarSharpIcon />}
+      {user.type === RoleType.Business && <BusinessIcon />}
+    </div>
```

Comment:
This could use a component

Then it would be simple to add e.g. hover that shows the role in text